data_hub_api/docmaps/v2/codecs/elife_manuscript.py

```diff
@@ -228,8 +228,19 @@ def get_docmap_elife_manuscript_output_for_published_step(
 
 
 def get_docmap_elife_manuscript_output_content_for_vor(
-    query_result_item: ApiInput
+    query_result_item: ApiInput,
+    vor_version_number: int
 ) -> Sequence[DocmapContent]:
+    if vor_version_number > 1:
```

Comment:
Doesn't that mean that version 1 would point to the article without version number. And that would usually redirect to the latest version?

Reply:
Double checking with Fred: https://github.com/elifesciences/data-hub-api/pull/177/files#r1661245261